bluedonkey.py

```diff
@@ -57,26 +65,35 @@ def init_filter():
 
 class dummy_car_control():
     def __init__(self):
+        ## Commented per jkridner's advice
         import car_control
         self.c = car_control.car_control()
+        
+        #Output for the status
+        self.status_port = 3004
+        self.status_out = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self.status_out.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+        self.status_out.connect(("", self.status_port))
+        self.status_file = self.status_out.makefile('w', buffering=None)
+        #pass
```

Comment:
Is this now a file or just a file handle?

Reply:
This section is filehandles pointing to sockets.